atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/EteSetup.java

```diff
@@ -51,8 +51,18 @@
 
     private static DockerComposeRule docker;
     private static List<String> availableClients;
+    private static Duration waitDuration;
 
     static RuleChain setupComposition(Class<?> eteClass, String composeFile, List<String> availableClientNames) {
+        return setupComposition(eteClass, composeFile, availableClientNames, Duration.TWO_MINUTES);
+    }
+
+    static RuleChain setupComposition(
+            Class<?> eteClass,
+            String composeFile,
+            List<String> availableClientNames,
+            Duration waitTime) {
+        waitDuration = waitTime;
```

Comment:
This fails if someone calls the below `setupComposition` directly. Try passing it in one level further.